HISTORY.rst

```diff
@@ -5,6 +5,8 @@ History
 Unreleased / master
 -------------------
 
+* ``ispyb.sqlalchemy.url()`` is a function that generates the SQLAlchemy connection URL from the ISPyB configuration
+
```

Comment:
Should we also include the `ispybl.sqlalchemy.session()` deprecation notice in the release notes?

Reply:
I'd say don't bother - it's not even a DeprecationWarning yet. Basically 'discourage usage until we figure out what we want to do exactly'. Change to DeprecationWarning with #125, and add to notes at that time?